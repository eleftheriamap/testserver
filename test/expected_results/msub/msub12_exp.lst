
test/exp/test/gen/msub/msub12_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	917b4339 	add	x25, x25, #0xed0, lsl #12
   4:	91536442 	add	x2, x2, #0x4d9, lsl #12
   8:	5154f2b5 	sub	w21, w21, #0x53c, lsl #12
   c:	9b02d73d 	msub	x29, x25, x2, x21
  10:	8a000000 	and	x0, x0, x0
