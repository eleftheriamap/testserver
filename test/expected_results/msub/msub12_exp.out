Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 00000000004d9000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 00000000ffac4000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000ed0000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = ffffb832afac4000
X30    = 0000000000000000
PC     = 0000000000000010
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x917b4339
0x00000004 : 0x91536442
0x00000008 : 0x5154f2b5
0x0000000c : 0x9b02d73d
0x00000010 : 0x8a000000
